docs/root/intro/arch_overview/security/bug_bounty.rst

```diff
@@ -0,0 +1,176 @@
+.. _arch_overview_bug_bounty:
+
+Bug bounty
+==========
+
+In the near future, Envoy will have a bug bounty program. This will be open to all security
+researchers and will provide rewards for vulnerabilities discovered and reported according
+to the rules below.
+
+.. note::
+  This is not yet funded, stay tuned and reach out to envoy-security@googlegroups.com for further
```

Comment:
should these come through g.co/vulnz? or only envoy-security@?

if envoy-security@, then there must be a staffed team able to triage these, which is a whole process in and off itself. but if we use g.co/vulnz you can use Google's.

Reply:
envoy-security@; we already have a staffed team that does triage and provides around 24H SLO. There are multiple Googlers on that list FWIW.